docs/getting-started.md

```diff
@@ -1,21 +1,162 @@
+
+When you enter a new narrow, Zulip Terminal normally will automatically place your cursor on the oldest unread message in that narrow.
+
+![Screenshot Showing Reading Messages](getting-started-imgs/read-messages.png)
+
+## Narrow to the Stream or Topic of a Message
+
+Now that we know how to read through messages, let's try filtering to see only messages related to the one we currently have selected. To see this in action you'll want to head back to **All Messages** (<kbd>a</kbd>) before each bullet point. 
+
++ Type <kbd>s</kbd> to narrow to the stream of the current message.
+
++ Type <kbd>S</kbd> to narrow to the topic of the current message. *Note that this works even if you weren't already in the topic's stream.*
+
+You can also type <kbd>z</kbd> to 'zoom in' or 'zoom out.' What do I mean by 'zoom in/out?' Essentially if you're currently in a topic narrow, <kbd>z</kbd> will place you in the selected message's stream, but if you're in a stream <kbd>z</kbd> will place you in the selected message's topic. Try it and see for yourself, press <kbd>z</kbd> a few times and check out your [current narrow](#Your-Current-Narrow).
+
+# Sending Messages
+
+## Reply to a Message
+
+To reply to a message in a narrow (stream/topic/PM), rest your cursor on the message you want to reply to and then type <kbd>r</kbd>. Type your message in the Message box that pops up at the bottom of the middle column. Type <kbd>ctrl</kbd><kbd>d</kbd> to send. If you change your mind and don't want to send the message, type <kbd>esc</kbd> to get out of the message editor. Let's try replying to a random message in the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream (don't worry about messing anything up, the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream was made for stuff like this).
+
+![Screenshot Showing Reply](getting-started-imgs/reply.png)
+
+### Thumbs-up Reaction
+
+If you want to show you agree with the current message, type <kbd>+</kbd> to add a thumbs-up reaction. Type <kbd>+</kbd> again to remove the thumbs-up.
+
+![Screenshot Showing Thumbs-up](getting-started-imgs/thumbs-up.png)
+
+## Send a PM
+
+To send a PM:
+1. Type <kbd>x</kbd> from anywhere in Zulip Terminal. A message editor will pop open at the bottom of the middle column. 
+2. Type in part of the name of the person you'd like to send a PM to (IMG 1)
+3. Press <kbd>ctrl</kbd><kbd>f</kbd> and a list of potential recipients will appear in the footer (highlighted in red). 
```

Comment:
> Press <kbd>ctrl</kbd><kbd>f</kbd> and a list of potential recipients will appear in the footer (highlighted in red).

I ran into an issue where this list didn't show up; the header just stayed like this:

![image](https://user-images.githubusercontent.com/22248748/109050885-ea73b680-76a7-11eb-88ca-25e627c33467.png)

But then I saw a recent [discussion](https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/PM.20compose.20bug.20%28bad.2Fno.20recipients%29.20.23T935/near/1123180) on CZO and thought I'd try grabbing the app from current `main` (instead of the latest released version) to see if the issue was fixed…

…and it was! 🎉 So, no problem here.

Reply:
Also, today I learned that you can use `<kbd>x</kbd>` and it'll render something that looks like a keyboard key: <kbd>x</kbd>. Neat, thanks for that!

Reply:
Do you think it's OK to have the tutorial be consistent with an unreleased version, then?

Reply:
> Do you think it's OK to have the tutorial be consistent with an unreleased version, then?

Hmm, good question. For purely developer-facing documentation (which is most or all of zulip-mobile's documentation, I guess), I wouldn't hesitate, because it's a safe assumption that developers are working on the latest `main` or at least can get there pretty easily (I used the [`Installation -> Latest` part of the README](https://github.com/zulip/zulip-terminal#installation)).

zulip/zulip.git keeps plenty of [user-facing documentation](https://github.com/zulip/zulip/tree/master/templates/zerver/help), so I looked there for some clues. I see a few commits (zulip/zulip@160cc5120, zulip/zulip@1d5aa2e51) where some functionality changes, and the user-facing doc is updated simultaneously. So, for some time, the user-facing doc on GitHub is inconsistent with any released version.

…but this turns out to not be a huge problem for that project, because the most natural way for a user to access the doc isn't through GitHub, but through the Zulip installation itself, where, if all goes to plan, the documentation will be precisely aligned with everything else. E.g., for CZO, you'd go to https://chat.zulip.org/help/; or, for the Recurse Center, https://recurse.zulipchat.com/help/.

In ZT's case, GitHub is the place a user goes for user-facing documentation like this tutorial, and GitHub naturally shows the latest code, which is often unreleased. Hmm.

I guess I don't have a definite answer. I suppose one solution would be to point users to a published website for user-facing documentation, and have it default to the latest stable release while allowing them to select something else. Like [React Native](https://reactnative.dev/docs/getting-started):

<img width=400 src="https://user-images.githubusercontent.com/22248748/109060834-e51c6900-76b3-11eb-9948-979799cd2f3d.png" />

But that seems like a lot of effort.

Reply:
Also, I guess, the more frequently releases get made, the less one has to think about an inconsistency like this being a problem.